packages/next/src/build/swc/index.ts

```diff
@@ -424,13 +424,17 @@ function loadNative(isCustomTurbopack = false) {
 
               let child = spawn(__INTERNAL_CUSTOM_TURBOPACK_BINARY, args, {
                 stdio: 'pipe',
-                env: {
-                  ...process.env,
-                },
               })
               child.on('message', (message: any) => {
-                console.log(message)
+                require('console').log(message)
               })
+              child.stdout.on('data', (data: any) => {
+                require('console').log(data.toString())
+              })
+              child.stderr.on('data', (data: any) => {
+                require('console').log(data.toString())
```

Comment:
```suggestion
                require('console').error(data.toString())
```
Are we not writing this back to stderr on purpose?

Reply:
not intentionally, this was just good enough to ensure dev instance boots up correctly. Since this is internal testing purpose only, I assume it won't be huge blocker to not to polish these behavior (as long as it works).